materials/sql_joins.md

```diff
@@ -7,44 +7,65 @@ language: SQL
 
 ### Basic join
 
+* [`JOIN`](http://www.w3schools.com/sql/sql_join.asp) 
+    * combine rows from multiple tables
+    * based on condition
+* This query selects `species` records in the `PortalMammals_main` table and 
+  `scientificname` found in the `PortalMammals_species` table.
+    * The query links the `species` with `new_code` from `PortalMammals_species`.
+  
 ```
-SELECT yr, mo, dy, scientificname
-FROM Main
-JOIN Species ON species = new_code;
+SELECT species, scientificname
+FROM PortalMammals_main
+JOIN PortalMammsls_species WHERE species = new_code;
```

Comment:
'Fortunately' then, using `WHERE` is only semantically wrong and doesn't break anything.
 
@gvwilson recently shared a teaching philosophy with me that suggests that sometimes the best learning strategy is to first show a bad example and then make the fixes to demonstrate the best practice. Maybe it matters where where the students are on the novice-expert shift.